server/api/teams.go

```diff
@@ -217,6 +222,7 @@ func (a *API) handleGetTeamUsers(w http.ResponseWriter, r *http.Request) {
 	userID := getUserID(r)
 	query := r.URL.Query()
 	searchQuery := query.Get("search")
+	excludeBots := r.URL.Query().Get("exclude_bots") == True
```

Comment:
Looking good.  It's unfortunate that the previous behaviour was to include bots, since I think most usage of this API will exclude bots and it would be nice to have that as the default (i.e. no param provided).

Reply:
@wiggin77 could we make that change in the v3 you're working on?